server/src/main/java/io/crate/protocols/ssl/SslContextProvider.java

```diff
@@ -169,15 +171,15 @@ private SslContext serverContext(Protocol protocol) {
             var trustStore = Optionals.of(() -> loadKeyStore(trustStorePath, trustStorePass));
             var trustManagers = trustStore
                 .map(SslContextProvider::createTrustManagers)
-                .orElseGet(() -> new TrustManager[0]);
+                .orElseGet(() -> SslContextProvider.createTrustManagers(null));
```

Comment:
My idea was that well known CA's are trusted but yeah can be dropped - If I got your point correctly, this somehow defeats idea of setting up proper HBA cert authentication on the server side, reverted for the server context

Reply:
> My idea was that well known CA's are trusted

This would mean pretty much anyone could create a client certificate that would be trusted by the server as long as they have control of the domain name of the client node.

For authentication via client certificate it is important that only certificates that _you_ supplied are trusted.